src/assets/styles/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$space-0: 0rem;
+$space-1: 0.063rem;
+$space-5: 0.313rem;
+$space-10: 0.625rem;
+$space-15: 0.938rem;
+$space-30: 1.875rem;
+$space-200: 12.5rem;
+$space-350: 21.875rem;
```

Comment:
As I suggested in a daily call the spaces should be 2, 4, 8, 12, 16, 20, 24...64

Rest of the numbers can be calculate from these. 

$sp-2: 
$sp-4: 
$sp-8: 
$sp-12: 
$sp-20:......!